src/components/settings-menu/NotificationsSection.tsx

```diff
@@ -1,122 +1,401 @@
+      }
+      leftComponent={
+        <Box
+          style={{
+            opacity: groupOff || !notificationSettings.enabled ? 0.25 : 1,
+          }}
+        >
+          {wallet?.image ? (
+            <ImageAvatar image={wallet?.image} size="smedium" />
+          ) : (
             <ContactAvatar
-              alignSelf="center"
-              color={colors.appleBlue}
-              marginRight={10}
+              color={wallet?.color}
               size="small"
-              value="notben.eth"
+              value={profileUtils.addressHashedEmoji(wallet?.address)}
             />
-          }
-          onPress={() => onPress('notben.eth')}
-          size={52}
-          titleComponent={<MenuItem.Title text="notben.eth" />}
-        />
-        <MenuItem
-          hasRightArrow
-          labelComponent={
-            <MenuItem.Label
-              text={`${lang.t(
-                'settings.notifications_section.received'
-              )}, ${lang.t('settings.notifications_section.sold')}, ${lang.t(
-                'settings.notifications_section.minted'
-              )}, ${lang.t('settings.notifications_section.plus_n_more', {
-                n: 4,
-              })}`}
-            />
-          }
-          leftComponent={
-            <ContactAvatar
-              alignSelf="center"
-              color={colors.red}
-              marginRight={10}
-              size="small"
-              value="pugson.eth"
-            />
-          }
-          onPress={() => onPress('pugson.eth')}
-          size={52}
-          titleComponent={<MenuItem.Title text="pugson.eth" />}
-        />
-      </Menu>
-      <Menu>
-        <MenuItem
-          disabled
-          rightComponent={<Switch />}
-          size={52}
-          titleComponent={
-            <MenuItem.Title
-              text={lang.t('settings.notifications_section.watched_wallets')}
-              weight="bold"
-            />
-          }
-        />
-        <MenuItem
-          hasRightArrow
-          labelComponent={
-            <MenuItem.Label
-              text={lang.t('settings.notifications_section.off')}
-            />
-          }
-          leftComponent={
-            <ContactAvatar
-              alignSelf="center"
-              color={colors.red}
-              marginRight={10}
-              size="small"
-              value="moxey.eth"
-            />
-          }
-          onPress={() => onPress('moxey.eth')}
-          size={52}
-          titleComponent={<MenuItem.Title text="moxey.eth" />}
-        />
-      </Menu>
-    </MenuContainer>
+          )}
+        </Box>
+      }
+      onPress={() => navigateToWalletSettings(walletName, wallet.address)}
+      size={52}
+      titleComponent={<MenuItem.Title text={walletName} />}
+    />
+  );
+};
+
+const NotificationsSection = () => {
+  const { navigate } = useNavigation();
+  const { colors } = useTheme();
+  const { chainId } = useAccountSettings();
+  const notMainnet = chainId !== 1;
```

Comment:
we have `isTestnetNetwork` for this, you can get the `network` from `useAccountSettings` above